docs/vm/resource-overcommit.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 7
+keywords:
+  - Harvester
+  - Overcommit
+  - Overprovision
+  - ballooning
+Description: Overcommit resources to a VM.
+---
+
+# Resource Overcommit
+
+supports global configuration of resource overload percentages on CPU, memory, and storage. By setting [`overcommit-config`](../settings/settings.md#overcommit-config), this will allow scheduling of additional virtual machines even when physical resources are fully utilized.
```

Comment:
`supports global configuration of resource overload percentages on CPU, memory, and storage`
Perhaps the sentence misses a noun?

Reply:
Added. Thanks!